src/components/activity-card/activity-card.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  ActivityBody,
+  RecentEventsBody,
+} from '@console/dynamic-plugin-sdk/provisional';
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import { FirehoseResource } from "@console/dynamic-plugin-sdk";
+import { IBM_STORAGE_CSI_PROVISIONER } from '../../constants/index';
+import { 
+   EventModel, 
+   StorageInstanceModel,   
+  } from '../../models';
+import './activity-card.scss';
+//import {GetFlashSystemResource} from '../../constants/resources';
+import { EventKind } from '../../types';
+
+const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+
+const RecentEvent = (props) =>{
+  //const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
+  //const name= loaded && !loadError? data?.[0]?.metadata.name: '';
+  //const namespace= loaded && !loadError? data?.[0]?.metadata.namespace: '';
```

Comment:
Lets  put this logic when required, We can avoid commenting code and rather opening issues/jira for the task that are pending

Reply:
Or if there is a dependency on another PR, we can say merge after <PR number>